src/create_gene_annotations.py

```diff
@@ -0,0 +1,91 @@
+import pandas as pd
+from bioservices import BioMart
+from io import StringIO
+from pathlib import Path
+from numpy import loadtxt
+
+ensembl_datasets = ["hsapiens_gene_ensembl", "mmusculus_gene_ensembl"]
+attributes_of_interest = [
+    "ensembl_gene_id",
+    "external_gene_name",
+    "chromosome_name",
+    "description",
+]
+
+for ds in ensembl_datasets:
+    # Establish a connection to ENSEMBL biomart and add the dataset to
+    # the XML query
+    s = BioMart()
+    s.add_dataset_to_xml(ds)
+
+    # Add each attribute to the XML query
+    for attribute in attributes_of_interest:
+        s.add_attribute_to_xml(attribute)
+
+    # Generate the XML query and store the result, convert to buffer
+    xml_query = s.get_xml()
+    result_string = s.query(xml_query)
+    result_buffer = StringIO(result_string)
+
+    # Use buffer to make a dataframe, where genes will be annotated
+    genes_df = pd.read_csv(result_buffer, sep="\t", names=attributes_of_interest)
```

Comment:
is this necessary? the result_string is like 30,000 entries in a CSV format, so i imagine it takes up a decent amount of memory, as does the result_buffer, but i have no clue